fda/basis.py

```diff
@@ -19,6 +19,7 @@
 
 from . import grid
 from .functional_data import FData, _list_of_arrays
+from .lfd import Lfd
```

Comment:
Although we have got used to the `Lfd` name, maybe you should try to coin a better class and module name for a future pull request. I should have mentioned it when you first added that class.